### DISCUSSION TITLE
Infinite Carousal

### DISCUSSION THREAD

**Discussion by sayanChaterjee** [twbs/bootstrap#40956] [Ideas]
Hey @julien-deramond I’d like to contribute an infinite scrolling horizontal carousel component that pauses on hover. Looking forward to your thoughts

**Comment by julien-deramond** [comment]
Just to clarify, when you refer to an 'infinite scrolling horizontal carousel,' is it the same as our [Autoplaying carousels](https://getbootstrap.com/docs/5.3/components/carousel/#autoplaying-carousels)? If so, I'd say it shouldn't pause on hover but instead use a play/pause button for accessibility, as highlighted in the information callout on that page.

**Reply by sayanChaterjee** [reply]
No I mean continuously moving cards that will pause on hover. It's like the marquee but applicable for cards and background images as well thereby creating a carousel.
